dist/tools/output_to_xunit/output_to_xunit.py

```diff
@@ -0,0 +1,125 @@
+#! /usr/bin/env python3
+import argparse
+import xml.etree.ElementTree as ET
+from datetime import datetime as DT
+
+
+TIME_FORMAT = "%Y%m%d %H:%M:%S.%f"
+XUNIT_OUT = "xunit.xml"
+
+args_parser = argparse.ArgumentParser(
+    description="Parse RobotFrameworks output.xml to xUnit format",
+    formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+)
+args_parser.add_argument(
+    "--output",
+    type=str,
+    help="path to file where the output should be written to",
+    default=XUNIT_OUT,
+)
+args_parser.add_argument(
+    "input", type=str, help="path to a file in RobotFramework output format"
+)
+args = args_parser.parse_args()
+
+tree = ET.parse(args.input)
+root = tree.getroot()
+
+suite = root.find("suite")
+testsuite = {
+    "name": suite.get("name"),
+    "skipped": 0,
+    "failures": 0,
+    "errors": 0,
+}
+
+starttime = DT.strptime(suite.find("status").get("starttime"), TIME_FORMAT)
+endtime = DT.strptime(suite.find("status").get("endtime"), TIME_FORMAT)
+suitetime = str((endtime - starttime).total_seconds())
+testsuite["time"] = suitetime
+
+testcases = list()
+child_suites = suite.findall("suite")
+for suite in child_suites:
+    for test in suite.findall("test"):
+        testcase = {
+            "classname": f"{testsuite['name']}.{suite.get('name')}",
+            "name": test.get("name"),
+            "failed": False,
+            "skipped": False,
+        }
+
+        starttime = DT.strptime(test.find("status").get("starttime"), TIME_FORMAT)
+        endtime = DT.strptime(test.find("status").get("endtime"), TIME_FORMAT)
+        testtime = str((endtime - starttime).total_seconds())
+        testcase["time"] = testtime
+
+        # check if testcase failed
+        if test.find("status").get("status") == "FAIL":
+            testcase["failed"] = True
+            testcase["failed_text"] = test.find("status").text
+            testsuite["failures"] += 1
+
+        # check if testcase skipped
+        tags = [tag.text for tag in test.findall("tags/tag")]
+        if "skip" in tags:
+            testcase["skipped"] = True
+            testcase["skipped_text"] = test.find("status").get("status")
+            testsuite["skipped"] += 1
+
+        testcase["records"] = list()
+        for record in test.findall("kw[@name='Record Property']"):
+            r = dict()
+            for e in record.iter("msg"):
+                text = e.text
+                if text.startswith("NAME:"):
+                    r["name"] = text[len("NAME: ") :]
+                elif text.startswith("VALUE:"):
+                    r["value"] = text[len("VALUE: ") :]
+            testcase["records"].append(r)
+        testcases.append(testcase)
+
+stats = root.find("statistics")
+total_all = stats.find("total/stat[.='All Tests']")
+total_critical = stats.find("total/stat[.='Critical Tests']")
```

Comment:
```suggestion
stats = root.find("statistics/total")
total_all = next(obj for obj in stats if obj.text == 'All Tests')
total_critical = next(obj for obj in stats if obj.text == 'Critical Tests')
```

Reply:
Tested working on both python 3.6 and 3.8 :+1: